students/synowiec_krzysztof/lesson_02_flow_control/queen_move.py

```diff
@@ -0,0 +1,20 @@
+def main():
+ startRow = int(input())
```

Comment:
Looks like there is only one space used for main() function.
Spacing should be the same for all files within a project. Standard practice is to use is 4 spaces.